libkbfs/conflict_resolver.go

```diff
@@ -2873,6 +2895,12 @@ func (cr *ConflictResolver) syncBlocks(ctx context.Context, lState *lockState,
 		}
 	}
 
+	defer func() {
```

Comment:
Can you do an audit of all other `doBlockPuts` calls to see if they need a defer of `cleanUpBlockState`, too? Looks like there's one in `finalizeGCOp`?